src/mapping/config/MappingConfiguration.cpp

```diff
@@ -243,10 +264,15 @@ void MappingConfiguration::xmlTagCallback(
       else if (strPrealloc == "off")
         preallocation = Preallocation::OFF;
     }
+    if (tag.hasAttribute(ATTR_VERTICES_PER_PARTITION)) {
+      verticesPerPartition = tag.getDoubleAttributeValue(ATTR_VERTICES_PER_PARTITION);
+      relativeOverlap      = tag.getDoubleAttributeValue(ATTR_RELATIVE_OVERLAP);
+      projectToInput       = tag.getBooleanAttributeValue(ATTR_PROJECT_TO_INPUT);
+    }
 
     RBFParameter rbfParameter;
     // Check valid combinations for the Gaussian RBF input
-    if (type == VALUE_RBF_GAUSSIAN || type == VALUE_RBF_INV_MULTIQUADRICS || type == VALUE_RBF_MULTIQUADRICS || type == VALUE_RBF_CTPS_C2 || type == VALUE_RBF_CPOLYNOMIAL_C0 || type == VALUE_RBF_CPOLYNOMIAL_C6 || type == VALUE_RBF_CPOLYNOMIAL_C2 || type == VALUE_RBF_CPOLYNOMIAL_C4) {
+    if (type == VALUE_RBF_GAUSSIAN || type == VALUE_RBF_INV_MULTIQUADRICS || type == VALUE_RBF_MULTIQUADRICS || type == VALUE_RBF_CTPS_C2 || type == VALUE_RBF_CPOLYNOMIAL_C0 || type == VALUE_RBF_CPOLYNOMIAL_C6 || type == VALUE_RBF_CPOLYNOMIAL_C2 || type == VALUE_RBF_CPOLYNOMIAL_C4 || type == VALUE_RBF_POU) {
```

Comment:
The partition of unity is a method, not a kind of RBF.

I understand that this is a quick-n-drity way of getting the feature into preCICE, but isn't this a bit too cumbersome?
Given that you hard-coded the actual RBF type, I would move the partition of unity into a completely separate tag. This will be a good starting point for the future refactoring #1417.

If you do, please consider moving the pou to a separate `createMapping` too.

Reply:
This is only an implementation to test the mapping itself. The MappingConfiguration will change once we converged in #1417 and the configuration implementation should be trivial then.